test/from-draft7/to-2019-09/schema.json

```diff
@@ -40,15 +40,15 @@
         "title": "Nested half matching schema",
         "from": {
             "$schema": "https://json-schema.org/draft/2020-12/schema",
-            "$defs": {
```

Comment:
Why are you replacing them with `definitions`? This is a 2020-12 schema, so you should use `$defs`